UI/styles.css

```diff
@@ -0,0 +1,197 @@
+*{
```

Comment:
I believe UI files do not belong to the Task 4

Reply:
Ok, got it. I'll create a new branch for this task from the master and close this PR.